complete/src/main/resources/static/app.js

```diff
@@ -18,100 +18,134 @@ function refreshTimeTable() {
+                            <small class="fas fa-trash"></small>
+                        </button>
+                        <h5 class="card-title mb-1">${props.lesson.subject}</h5>
+                        <p class="card-text text-muted ml-2 mb-1">by ${props.lesson.teacher}</p>
+                        <small class="ml-2 mt-1 card-text text-muted align-bottom float-right">${props.lesson.id}</small>
+                        <p class="card-text ml-2">${lesson.studentGroup}</p>
+                    </div>
+                </div>
+            `));
             if (lesson.timeslot == null || lesson.room == null) {
                 unassignedLessons.append(lessonElement);
             } else {
-                $("#timeslot" + lesson.timeslot.id + "room" + lesson.room.id).append(lessonElement);
-                var lessonElementWithoutDelete = $("<div class=\"card lesson\" style=\"background-color: " + color +"\"><div class=\"card-body p-2\">"
-                        + "<h5 class=\"card-title mb-1\">" + lesson.subject + "</h5>"
-                        + "<p class=\"card-text text-muted ml-2 mb-1\">by " + lesson.teacher + "</p>"
-                        + "<small class=\"ml-2 mt-1 card-text text-muted align-bottom float-right\">" + lesson.id + "</small>"
-                        + "<p class=\"card-text ml-2\">" + lesson.studentGroup + "</p>"
-                        + "</div></div>");
-                $("#timeslot" + lesson.timeslot.id + "teacher" + convertToId(lesson.teacher)).append(lessonElementWithoutDelete.clone());
-                $("#timeslot" + lesson.timeslot.id + "studentGroup" + convertToId(lesson.studentGroup)).append(lessonElementWithoutDelete.clone());
+                $(`#timeslot${lesson.timeslot.id}room${lesson.room.id}`).append(lessonElement);
+                var lessonElementWithoutDelete = $(template({ lesson }, props => `
+                    <div class="card lesson" style="background-color: ${color}">
+                        <div class="card-body p-2">
+                            <h5 class="card-title mb-1">${props.lesson.subject}</h5>
+                            <p class="card-text text-muted ml-2 mb-1">by ${props.lesson.teacher}</p>
+                            <small class="ml-2 mt-1 card-text text-muted align-bottom float-right">${props.lesson.id}</small>
+                            <p class="card-text ml-2">${props.lesson.studentGroup}</p>
+                        </div>
+                    </div>
+                `));
+                $(`#timeslot${lesson.timeslot.id}teacher${convertToId(lesson.teacher)}`).append(lessonElementWithoutDelete.clone());
+                $(`#timeslot${lesson.timeslot.id}studentGroup${convertToId(lesson.studentGroup)}`).append(lessonElementWithoutDelete.clone());
             }
-            $("#deleteLessonButton-" + lesson.id).click(function() {
+            $(`#${id}`).click(() => {
                 deleteLesson(lesson);
             });
```

Comment:
Can we simplify this to something like this? (no idea)
` $(`#${id}`).click(() => deleteLesson(lesson));`

Reply:
Yes; done